ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss

```diff
@@ -357,7 +357,7 @@ $mobile-breakpoint: uswds-mobile-breakpoint.$theme-header-min-width - 1px;
       height: 100vh;
       left: 0;
       max-width: 100vw;
-      padding: $cbp-size-4;
+      padding: 32px; // 0 $cbp-size-4;
```

Comment:
@dannyk3941 Is there no pixel token for 32px?

Reply:
Resolved. The sizing tokens will be replaced once they are determined by the team.